recipes/nvcloth/1.1.6/conanfile.py

```diff
@@ -0,0 +1,194 @@
+import os
+import shutil
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import msvc_runtime_flag
+
+required_conan_version = ">=1.35.0"
+
+class NvclothConan(ConanFile):
+    name = "nvcloth"
+    license = "Nvidia Source Code License (1-Way Commercial)"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/NVIDIAGameWorks/NvCloth"
+    description = "NvCloth is a library that provides low level access to a cloth solver designed for realtime interactive applications."
+    topics = ("physics", "physics-engine", "physics-simulation", "game-development", "cuda")
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "use_cuda": [True, False],
+        "use_dx11": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": False,
+        "use_cuda": False,
+        "use_dx11": False
+    }
+
+    no_copy_source = True
+    short_paths = True
+
+    generators = "cmake"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+    
+    @property
+    def _is_msvc(self):
+        return str(self.settings.compiler) in ["Visual Studio", "msvc"]
+
```

Comment:
```suggestion
```

Deprecated. Prefer https://docs.conan.io/en/latest/reference/conanfile/tools/microsoft.html#conan-tools-microsoft-is-msvc